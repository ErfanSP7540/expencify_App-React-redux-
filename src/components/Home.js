import React from 'react'
import ExpenseList from './ExpenseList'
import ItemConnected from './ExpenseListItem'
import ExpenseslistFilter from './ExpenseListFilter'
import ExpenseListFilter from './ExpenseListFilter';

const Home = (props) => (
    <div>
        HOme Page
        <ExpenseListFilter />
        <ExpenseList/>
    </div>
)

export default Home;