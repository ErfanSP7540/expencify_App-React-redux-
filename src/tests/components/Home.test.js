import React from 'react'
import Home from '../../components/Home'

import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import  Adapter  from 'enzyme-adapter-react-16'
configure({ adapter:new Adapter()});



test('HOME PAGE TESTING SnapShot',()=>{
    
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
})