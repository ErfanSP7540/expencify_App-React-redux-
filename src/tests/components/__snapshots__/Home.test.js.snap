// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HOME PAGE TESTING SnapShot 1`] = `
<div>
  HOme Page
  <Connect(ExpenseslistFilter) />
  <Connect(ExpenseList) />
</div>
`;
